huaweicloud/services/bms/resource_huaweicloud_bms_instance.go

```diff
@@ -379,50 +380,53 @@ func resourceBmsInstanceRead(_ context.Context, d *schema.ResourceData, meta int
 		return nil
 	}
 
-	logp.Printf("[DEBUG] Retrieved Server %s: %+v", d.Id(), server)
-
-	d.Set("region", region)
-	d.Set("name", server.Name)
-	d.Set("image_id", server.Image.ID)
-	d.Set("flavor_id", server.Flavor.ID)
-	d.Set("host_id", server.HostID)
+	log.Printf("[DEBUG] Retrieved Server %s: %+v", d.Id(), server)
+	mErr := multierror.Append(nil,
```

Comment:
please try to merge the setting operations

Reply:
ok，it was updated